src/commands/reactionCommands.ts

```diff
@@ -15,7 +15,7 @@ export const googleReact: ReactionCommand = {
             return;
         }
         const resultantContent = replaceMentions(message);
-        message.channel.send(`https://www.google.com/search?q=${encodeURIComponent(resultantContent)}`);
+        message.reply(`https://www.google.com/search?q=${encodeURIComponent(resultantContent)}`);
```

Comment:
```suggestion
        message.reply({
            content: `https://www.google.com/search?q=${encodeURIComponent(resultantContent)}`,
            allowedMentions: {
                repliedUser: false
            }
        });
```
Thinking about this...

I want the person who reacted to have the context (so the reply makes sense) but I don't want the person who wrote the initial googleable object to be notified if someone reacts to google it.

https://discordjs.guide/additional-info/changes-in-v13.html#replies-message-reply

Thoughts?